aws/terminator/paas.py

```diff
@@ -117,3 +117,131 @@ def terminate(self):
             else:
                 # delete streaming distribution
                 self.client.delete_streaming_distribution(Id=self.Id, IfMatch=ETag)
+
+
+class Ecs(DbTerminator):
+    @property
+    def age_limit(self):
+        return datetime.timedelta(minutes=20)
```

Comment:
The current import causes this to fail.  I'd suggest this change earlier in the file:
```
+++ b/aws/terminator/paas.py
@@ -1,4 +1,4 @@
-from datetime import datetime
+import datetime
 
 from . import DbTerminator, Terminator
 
@@ -39,7 +39,7 @@ class LambdaLayers(Terminator):
 
     @property
     def created_time(self):
-        return datetime.strptime(self.instance['LatestMatchingVersion']['CreatedDate'], "%Y-%m-%dT%H:%M:%S.%f%z")
+        return datetime.datetime.strptime(self.instance['LatestMatchingVersion']['CreatedDate'], "%Y-%m-%dT%H:%M:%S.%f%z")
 
     def terminate(self):
         for version in self.client.list_layer_versions(LayerName=self.name)['LayerVersions']:
```